packages/neon-core/__tests__/tx/calculate.ts

```diff
@@ -42,12 +48,15 @@ describe("calculateInputs", () => {
       assets: {
         NEO: {
           unspent: [{ value: 10, txid: "ab", index: 0 }],
+          hash: NEO_ASSET_HASH,
         },
         GAS: {
           unspent: [{ value: 1.5, txid: "cd", index: 0 }],
+          hash: GAS_ASSET_HASH,
         },
         TEST: {
           unspent: [{ value: 1, txid: "ef", index: 0 }],
+          hash: NEO_ASSET_HASH,

```

Comment:
Why does the TEST asset have the the NEO_ASSET_HASH?

Reply:
we're going to need a valid hash to test, I thought I'd use the NEO hash as a standard hash, but it could be another one

Reply:
This is a unit test so just generate a random hash would do.

Reply:
ok, I will do it